examples/with-remix-thirdpartyemailpassword/app/routes/_index.tsx

```diff
@@ -0,0 +1,181 @@
+import {
+  CelebrateIcon,
+  SeparatorLine,
+  BlogsIcon,
+  GuideIcon,
+  SignOutIcon,
+} from "../../assets/images";
+import { signOut } from "supertokens-auth-react/recipe/thirdpartyemailpassword/index.js";
+import { recipeDetails } from "../config/frontend";
+import SuperTokens from "supertokens-auth-react";
+import { LoaderFunctionArgs, redirect } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import { SessionDataForUI } from "../lib/superTokensTypes";
+import { SessionContainerInterface } from "supertokens-node/lib/build/recipe/session/types";
+import { getSessionDetails } from "../lib/superTokensHelpers";
+import { TryRefreshComponent } from "../components/tryRefreshClientComponent";
+import { SessionAuthForRemix } from "../components/sessionAuthForRemix";
+
+export async function loader({ request }: LoaderFunctionArgs): Promise<{
+  session: SessionContainerInterface | undefined;
+  hasInvalidClaims: boolean;
+  hasToken: boolean;
+  nextResponse: Response | null;
+}> {
+  try {
+    const { session, hasInvalidClaims, hasToken, nextResponse } =
+      await getSessionDetails(request);
+    console.log("does the user have invalid claims?", hasInvalidClaims);
+    console.log("does the user have an access token??", hasToken);
+    if (session) {
+      console.log("there is an active session");;
+    }
+    if (!session) {
+      console.log("session does not exist or has expired");
+    }
+    if (nextResponse) {
+      return {
+        session,
+        hasInvalidClaims,
+        hasToken,
+        nextResponse,
+      };
+    } else {
+      return {
+        session,
+        hasInvalidClaims,
+        hasToken,
+        nextResponse: null,
+      };
+    }
+  } catch (error) {
+    // throw new Error('');
+    console.error("Error retrieving session:", error);
+    throw error;
+  }
+}
+
+export default function Home() {
+  const loaderData = useLoaderData<{
+    session: SessionContainerInterface | undefined;
+    hasInvalidClaims: boolean;
+    hasToken: boolean;
+    nextResponse: Response | null;
+  }>();
+
+  if (loaderData.nextResponse) {
+    return (
+      <div>
+        Something went wrong while trying to get the session. Error - 
+        {loaderData.nextResponse.status} {loaderData.nextResponse.statusText}
+      </div>
+    );
+  }
+
```

Comment:
In the logic below, after logging in, deleting the access token and refreshing always redirects to `/auth` instead of refreshing the session.